zulipterminal/ui.py

```diff
@@ -77,6 +77,24 @@ def set_footer_text(self, text_list: Optional[List[Any]]=None,
             time.sleep(duration)
             self.set_footer_text()
 
+    @asynch
+    def set_typeahead_footer(self, suggestions: List[Any],
+                             state: Optional[int],
+                             is_truncated: bool) -> None:
+        # Wrap by space.
+        suggestions = list(map(lambda x: ' ' + x + ' ', suggestions))
+
+        if suggestions:
+            if state is not None:
+                suggestions[state] = ('code', suggestions[state])
+            if is_truncated:
+                suggestions += [' [more] ']
+            suggestions.insert(0, [' '])  # Add leading space.
+        else:
+            suggestions = [' [No matches found]']
+
+        self.set_footer_text(suggestions)
```

Comment:
minor: To avoid the footer getting 'stuck' before the next commit, we could just use the time parameter here?

Reply:
Did this, but if `ctrl f` is pressed 10 times, the help text will change 10 times. I guess we could let this slide since it's fixed in the next commit.